pom.xml

```diff
@@ -21,6 +27,19 @@
         <tag>HEAD</tag>
     </scm>
 
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <organization>
+      <name>FeedHenry by Red Hat</name>
```

Comment:
Let's be consistent. IF we are going with Red Hat in the NOTICE file, then let's use it here too.